sql/updates/world/master/9999_99_99_19345_world.sql

```diff
@@ -2,6 +2,6 @@ DELETE FROM `spell_areatrigger` WHERE (`SpellMiscId`=5420 AND `AreaTriggerId`=10
 INSERT INTO `spell_areatrigger` (`SpellMiscId`, `AreaTriggerId`, `MoveCurveId`, `ScaleCurveId`, `MorphCurveId`, `FacingCurveId`, `DecalPropertiesId`, `TimeToTarget`, `TimeToTargetScale`, `VerifiedBuild`) VALUES
 (5420, 10133, 0, 0, 0, 0, 0, 0, 7177, 23420); -- SpellId : 5740
 
-DELETE FROM `spell_script_names` WHERE `spell_id`=5740;
+DELETE FROM `spell_script_names` WHERE `spell_id`=5740 OR `ScriptName`="spell_warl_rain_of_fire";
```

Comment:
Yes, that is taken care of already in commit https://github.com/TrinityCore/TrinityCore/pull/19345/commits/42591e59c87f441f3c9d91fa86dd00437bc9aeac. Notice the use of `OR` as a divider.
Deleting any spell scripts using `spell_id` 5740 is just an extra precaution. :-)